test/accessibility/apps/visualize.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'visualize', 'header']);
+  const a11y = getService('a11y');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('Visualize', () => {
+    const vizName = 'Visualize Listing A11y';
+    before(async () => {
+      // await esArchiver.loadIfNeeded('logstash_functional');
+      // await kibanaServer.uiSettings.update({
+      //   defaultIndex: 'logstash-*',
+      // });
+      // //await esArchiver.load('visualize');
+      // await PageObjects.visualize.gotoVisualizationLandingPage();
+      // await PageObjects.visualize.deleteAllVisualizations();
+      await PageObjects.common.navigateToApp('visualize');
+    });
+
+    it('visualize', async () => {
+      await a11y.testAppSnapshot();
+    });
+
+    it('click on create visualize wizard', async () => {
+      await PageObjects.visualize.navigateToNewVisualization();
+      await a11y.testAppSnapshot();
+    });
+
+    // it('create visualize button', async () => {
```

Comment:
Can we `skip` this test instead of commenting it out so that it's clear we intent to implement?

```suggestion
    it.skip('create visualize button', async () => {
```